src/common/src/config.rs

```diff
@@ -216,9 +225,8 @@ mod default {
         201326592
```

Comment:
Also rename this? 😁

Reply:
This config is not used by any component, I've already removed it from the config struct.